packages/@ember/-internals/metal/lib/property_get.ts

```diff
@@ -172,11 +171,3 @@ _getProp({ unknownProperty() {} }, 1 as any);
 
 get({}, 'foo');
 get({}, 'foo.bar');
-
-let fakeProxy = {} as ProxyMixin<unknown>;
-setProxy(fakeProxy);
```

Comment:
I'm not sure we want to merge this, but it's necessary to avoid circularity that causes build issues.

Reply:
@pzuraq can you add some color on why this was added, and how important it is for perf for these?

Reply:
@wagenet what's the circularity issue it causes?

Reply:
when using `get` with a proxy it goes through a different code path, so this was attempting to warm that up. Unsure if it was really that impactful, and it will be less impactful if proxy usage has dropped (which was the goal at the time)